nx.json

```diff
@@ -21,5 +21,11 @@
     "test": {
       "outputs": ["{projectRoot}/coverage"]
     }
+  },
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "namedInputs": {
```

Comment:
This configuration is related to how the nx cache works and when it should be invalidated, [see](https://nx.dev/concepts/more-concepts/customizing-inputs) Given that we had issues with the cache in the past, we should take a look at this in the future.